src/features/common/Layout/Footer/index.tsx

```diff
@@ -75,6 +65,10 @@ export default function Footer() {
         countryCode = localStorage.getItem('countryCode');
         if (countryCode) setSelectedCountry(countryCode);
       }
+      if (localStorage.getItem('language')) {
+        langCode = localStorage.getItem('language');
+        if (langCode) setLanguage(langCode);
```

Comment:
Loading from local storage as other wise the language shown in the footer is the default `English` if i18n was not yet initialized.